src/app/components/global-lists/global-lists.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import { NamedEntitiesList } from '../../models/evt-models';
+import { NamedEntitiesParserService } from '../../services/xml-parsers/named-entities-parser.service';
+
+@Component({
+  selector: 'evt-global-lists',
+  templateUrl: './global-lists.component.html',
+  styleUrls: ['./global-lists.component.scss'],
+})
+export class GlobalListsComponent {
+  lists$: Observable<GlobalList[]> = this.neParserService.namedEntities$.pipe(
+    map(ne => (ne.persons.lists.concat(ne.places.lists, ne.organizations.lists, ne.events.lists))),
+    map(lists => (lists.map(list => ({
+      ...list,
+      icon: this.getIconByType(list),
+    })))),
+    tap(lists => {
+      if (!this.selectedList && lists[0]) {
+        this.openList(undefined, lists[0]);
+      }
+    }),
+  );
+  selectedList: NamedEntitiesList;
+
+  constructor(
+    private neParserService: NamedEntitiesParserService,
+  ) {
+  }
+
+  openList(event: MouseEvent, list: NamedEntitiesList) {
+    if (event) {
+      event.stopPropagation();
+    }
+    if (this.selectedList !== list) {
+      this.selectedList = list;
+    }
+    // else {
+    //   this.toggleListSublists(event, list);
+    // }
+  }
+
+  private getIconByType(list: NamedEntitiesList) {
+    switch (list.namedEntityType) {
+      case 'person':
+        return 'user';
+      case 'place':
+        return 'map-marker';
+      case 'org':
+        return 'users';
+      case 'event':
+        return 'calendar';
+      default:
+        return 'list';
+    }
+  }
+}
+
+interface GlobalList extends NamedEntitiesList {
+  icon: string;
+}
```

Comment:
Shouldn't this be inside `models.ts`

Reply:
It's strictly connected to the component and useful only in it.